bindings/CXX11/adios2/cxx11/KokkosView.h

```diff
@@ -17,24 +17,24 @@ struct memspace_kokkos_to_adios2<Kokkos::HostSpace>
     static constexpr adios2::MemorySpace value = adios2::MemorySpace::Host;
 };
 
-#if defined(KOKKOS_ENABLE_CUDA) && defined(ADIOS2_HAVE_CUDA)
+#if define(KOKKOS_ENABLE_CUDA) || define(ADIOS2_HAVE_CUDA)
```

Comment:
Is define() rather than defined() a valid variant in the preprocessor?

Reply:
Actually #if defined is valued:
#define ABC 2
#include <iostream>

int main()
{

#if defined ABC
std::cout << "1: yes\n";
#else
std::cout << "1: no\n";
#endif
}